openapi/multi-yaml/components/schemas/Checkout.yaml

```diff
@@ -76,3 +77,9 @@ properties:
     type: string
     format: date-time
     example: '2024-01-01T12:00:00Z'
+  completions: 
+    type: array
+    description: list of checkout completion attempts
+    items:
+      $ref: ./CheckoutCompletionAttempt.yaml
```

Comment:
I created this extra CheckoutCompletionAttempt.yaml file to add here because in this case the checkout completion should not include a nested checkout.
(For checkout completion events the checkout is nested inside the completion, for checkout completions in this completions array the checkout is not nested.)